src/spyglass/spikesorting/spikesorting_merge.py

```diff
@@ -68,6 +150,27 @@ def get_sorting(cls, key):
         query = source_table & cls.merge_get_part(key)
         return query.get_sorting(query.fetch("KEY"))
 
+    @classmethod
+    def get_sort_group_info(cls, key):
+        """get the sort group info associated with a spike sorting output
+        (e.g. electrode location, brain region, etc.)
+        Parameters:
+        -----------
+        key : dict
+            dictionary specifying the restriction (note: multi-source not currently supported)
+        Returns:
+        -------
+        sort_group_info : Table
+            Table linking a merge id to information about the electrode group.
+        """
+        source_table = source_class_dict[
```

Comment:
`source_class_dict` was later added as a property of the `Merge` class such that this method could `cls.source_class_dict` and remove the variable at the top of the schema